package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@phosphor-icons/react": "^2.0.10",
     "framer-motion": "^10.16.0",
+    "million": "2.6.0-beta.10",
```

Comment:
We're not using beta versions of million js

Reply:
Blame pnpm. `pnpm i million` installs the beta by default, for some reason.

Reply:
imagine using pnpm LOL